ilan/build_all

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+###########################################################################
+# (C) Copyright IBM Corporation 2016.                                     #
+#                                                                         #
+# Licensed under the Apache License, Version 2.0 (the "License");         #
+# you may not use this file except in compliance with the License.        #
+# You may obtain a copy of the License at                                 #
+#                                                                         #
+#      http://www.apache.org/licenses/LICENSE-2.0                         #
+#                                                                         #
+# Unless required by applicable law or agreed to in writing, software     #
+# distributed under the License is distributed on an "AS IS" BASIS,       #
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.#
+# See the License for the specific language governing permissions and     #
+# limitations under the License.                                          #
+###########################################################################
+
+USERNAME=$1
+PASSWORD=$2
+
+
+tmp=$(cat versions.csv| cut -d, -f1)
+
+versions=$(echo  $tmp | cut -d " " -f 2-)
```

Comment:
This was done to remove the word "version" from the list . I am getting the first column from the file which contains the word version as the beginning entry.